### ISSUE TITLE
Is there any plan to rewrite this app to Kotlin ?

### ISSUE THREAD

**Issue by Goooler** [ikarus23/MifareClassicTool#372]
If sure, I can provide help.

**Comment by ikarus23** [comment]
Thanks for offering your help. Sure a rewrite would be great. I mean in general. The code is old and could be improved very much by a skilled developer. But I'm no skilled developer. Also, this project is just one of my hobbies. I do it in my free time.

As visible in the commit history I don't have put much time into the app lately. I don't expect this to change dramatically. There is just to much other stuff.

So TLDR; I have no plans to rewrite this app in Kotlin.

**Comment by Goooler** [comment]
Alright.
